pkg/autodetect/main.go

```diff
@@ -109,8 +111,29 @@ func (b *Background) autoDetectCapabilities() {
 	b.cleanDeployments(ctx)
 }
 
+func (b *Background) isInListenedGroups(group *metav1.APIGroup) bool {
+	for _, groupName := range groupsListened {
+		if group.Name == groupName {
+			return true
+		}
+	}
+	return false
+}
+
 func (b *Background) availableAPIs(_ context.Context) ([]*metav1.APIResourceList, error) {
-	return b.dcl.ServerPreferredResources()
+	apiLists := []*metav1.APIResourceList{}
+	groupList, _ := b.dcl.ServerGroups()
+	for _, sg := range groupList.Groups {
+		if b.isInListenedGroups(&sg) {
+			groupAPIList, err := b.dcl.ServerResourcesForGroupVersion(sg.PreferredVersion.GroupVersion)
+			if err != nil {
+				return apiLists, err
```

Comment:
I would either return empty `apiList` or accumulate errors and return the error with gathered groups once all calls are done.